CorsixTH/Lua/objects/reception_desk.lua

```diff
@@ -92,48 +92,74 @@ function ReceptionDesk:onClick(ui, button)
 end
 
 function ReceptionDesk:tick()
+  --! Handle a patient at the reception desk.
+  --!param patient The patient at the front of queue.
+  --!param new (bool) Have they just entered the hospital?
+  local function handlePatient(patient, new)
+    if not new then
+      -- Patient redirected to reception, send them on their way
+      local id = patient.next_room_to_visit.room_info.id
+      return patient:queueAction(SeekRoomAction(id))
```

Comment:
Here a value is returned while in the other cases it is not. There is also no lua doc comment in the function docs.

I would suggest to make things consistent.